### ISSUE TITLE
AttributeError: 'CompositeField4' object has no attribute 'upsample_stride'

### ISSUE THREAD

**Issue by RM-8vt13r** [openpifpaf/openpifpaf#582]
When I try to use the openpifpaf package with a tshufflenetv2k30 backbone and trackingpose decoder, I get an error.

My system:
Windows 10 Pro 22H2, CUDA 12.0, Python 3.10.8

My environment:
```
certifi            2022.9.24
charset-normalizer 2.1.1
colorama           0.4.6
contourpy          1.0.6
cycler             0.11.0
ffmpeg             1.4
fonttools          4.38.0
idna               3.4
importlib-metadata 5.0.0
kiwisolver         1.4.4
matplotlib         3.6.2
numpy              1.23.4
opencv-python      4.6.0.66
openpifpaf         0.13.7
packaging          21.3
Pillow             9.3.0
pip                22.2.2
pyparsing          3.0.9
pysparkling        0.6.2
python-dateutil    2.8.2
python-json-logger 2.0.4
pytz               2022.6
requests           2.28.1
setuptools         63.2.0
six                1.16.0
torch              1.13.0+cu117
torchvision        0.14.0+cu117
tqdm               4.64.1
typing_extensions  4.4.0
urllib3            1.26.12
zipp               3.10.0
```

MWE:
```python
# imports
import openpifpaf
import openpifpaf.stream
import openpifpaf.predictor
import openpifpaf.visualizer
import openpifpaf.decoder
import argparse
import json

if __name__=="__main__":
    # Settings
    video_path = "./path/to/video.mkv"

    # Configure the decoder
    decoder_parser = argparse.ArgumentParser() # Initialize argparse
    openpifpaf.decoder.cli(decoder_parser) # Fill all decoder fields with their default values
    decoder_args = decoder_parser.parse_args(['--decoder', 'trackingpose:0']) # Use trackingpose decoder
    openpifpaf.decoder.configure(decoder_args)

    # Create predictor
    openpifpaf.predictor.Predictor.loader_workers = 1
    predictor = openpifpaf.predictor.Predictor(checkpoint='tshufflenetv2k30', json_data=False, visualize_image=False)

    # Create stream
    capture = openpifpaf.stream.Stream(video_path, preprocess=predictor.preprocess)

    # Forward pass
    for (preds, _, meta) in predictor.dataset(capture):
        pass
```

Expected behaviour:
the program loops over the video frames and makes predictions

Actual behaviour:
`predictor.dataset(capture)` raises an error `AttributeError: 'CompositeField4' object has no attribute 'upsample_stride'`

What I've tried:
I checked that `predictor` indeed contains `CompositeField4` modules, that lack some fields their constructor should have initialized. After testing I found out that the constructor of `CompositeField4` is never called. My theory is that the posetrack decoder was compiled with an old version of `CompositeField4`, and does therefore not work with the newest release of OpenPifPaf.

How can I fix this?

**Comment by svenkreiss** [comment]
Thanks for reporting this. I am exploring a fix in #583 .

**Comment by RM-8vt13r** [comment]
Thanks! I tested it today, and it works like a charm
